src/main/java/com/cgdp/library/service/AuthorService.java

```diff
@@ -21,4 +21,9 @@ public AuthorDTO createAuthor(String authorName) {
         AuthorEntity savedAuthor = authorRepository.save(authorEntity);
         return new AuthorDTO(savedAuthor.getId(), savedAuthor.getName());
     }
+
+    public AuthorEntity findById(Long authorId) {
```

Comment:
There are conventions that we should follow

When the method has prefix `find` it means that we don't know if what we are trying to find exists, so we should return an optional

When we know that something exists then the method should have prefix `get` and we should throw exception if whatever we are trying to get doesn't exist